### PR TITLE
Improvements to target tagging

### PR DESCRIPTION
**Pull request by doanac**
This changeset adds some usability improvements to how targets are tagged.

### CODE REVIEW

**Comment by doanac** [comment]
@mike-scott @Tim-Anderson - I added a `--dryrun` option to this command as well. I think this option is actually more powerful/useful than the `--append` I added.

File: subcommands/targets/tag.go [hunk 1]
```diff
@@ -32,33 +35,55 @@ func init() {
 	}
 	cmd.AddCommand(tagCmd)
 	tagCmd.Flags().StringVarP(&tagTags, "tags", "T", "", "comma,separate,list")
+	tagCmd.Flags().BoolVarP(&tagAppend, "append", "", false, "Append the given tags rather than set them")
```
Comment (Tim-Anderson) [inline subcommands/targets/tag.go:38]: should this be called appendtag or append_tag to make it clearer?

**Review by Tim-Anderson — COMMENTED** [review]
still reviewing

File: subcommands/targets/tag.go (hunk 1 above)
Comment (doanac) [reply subcommands/targets/tag.go:38]: I didn't have a strong opinion, but it felt redundant. Its the *tag* command. Feels a bit like saying "ATM machine", but some people prefer that.

Comment (Tim-Anderson) [reply subcommands/targets/tag.go:38]: we can let Mike chime in on that

Comment (mike-scott) [reply subcommands/targets/tag.go:38]: @Tim-Anderson the full command would look like this:
```
fioctl targets tag --append <tag> --by-version 42
```

When I read that back to myself it makes perfect sense.

**Review by mike-scott — APPROVED** [review]
Approach looks good to me.  Haven't had time to play test it.

**Pull request merged**